sonnet/src/nets/dnc/util_test.py

```diff
@@ -151,9 +151,15 @@ def testDifferentOutputSizeBreaks(self):
     lin_b = linear.Linear(output_size_b, name='lin_b')
     input_a = tf.random.uniform([batch_size, input_size])
     input_b = tf.random.uniform([batch_size, input_size])
-    with self.assertRaisesIncompatibleShapesError(
-        tf.errors.InvalidArgumentError):
-      util.apply_linear((input_a, input_b), (lin_a, lin_b))
+    if tf.__version__>="2.5.0":
```

Comment:
in general we only maintain compatibility with the latest release of TensorFlow 2, so I would prefer not to merge the change in this file.

Reply:
Hi @tomhennigan , Thank you for the review. Will revert this to the original version.